### PR TITLE
Remove feature flag guard around source archive jump-to-definition

### PR DESCRIPTION
**Pull request by jcreedcmu**
## Checklist

- [x] [CHANGELOG.md](https://github.com/github/vscode-codeql/blob/master/extensions/ql-vscode/CHANGELOG.md) has been updated to incorporate all user visible changes made by this pull request.
- [x] `@github/product-docs-dsp` has been notified, we probably want to wait on https://github.com/github/semmle-docs/issues/5 before merging this.

### CODE REVIEW

File: extensions/ql-vscode/src/config.ts [hunk 1]
```diff
@@ -42,13 +42,11 @@ const ROOT_SETTING = new Setting('codeQL');
 // Enable experimental features
 
 /**
- * This setting is deliberately not in package.json so that it does
+ * Any settings below deliberately not in package.json so that they do
```
Comment (aeisenberg) [inline extensions/ql-vscode/src/config.ts:45]: Nit: there's a grammatical error here.

**Comment by shati-patel** [comment]
> `@github/product-docs-dsp` has been notified, we probably want to wait on [github/semmle-docs#5](https://github.com/github/semmle-docs/issues/5) before merging this.

Docs update is done! We can publish that any time, just let me know when you plan to release.

File: extensions/ql-vscode/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
 # CodeQL for Visual Studio Code: Changelog
 
+## 1.2.0
+
+- Add CodeQL-query-powered handlers for 'Go to Definition' and 'Go To
+  References' on source archive files.
```
[the remark this answers is not in the document]
Comment (jcreedcmu) [reply extensions/ql-vscode/CHANGELOG.md:6]: Ok, sounds good! That change of emphasis makes sense now that you point it out.

File: extensions/ql-vscode/src/config.ts (hunk 1 above)
Comment (jcreedcmu) [reply extensions/ql-vscode/src/config.ts:45]: oops fixed

**Comment by jcreedcmu** [comment]
Should wait until after 1.1.5 release before merging this, just so I can rebase onto the CHANGELOG changes.

**Pull request merged**